server/src/main/java/com/ibm/ws/lars/rest/mongo/PersistenceBean.java

```diff
@@ -87,6 +88,7 @@
     private static final String ID = "_id";
 
     @Inject
+    @Injectable
     private com.mongodb.DB db;
 
```

Comment:
Are you _sure_ this is needed here? It's really odd to have a test framework annotation in your production code.

Reply:
I was led to believe this was needed to allow for the reflection fix to Deencapsulation.setField() being deprecated - will push an updated PR with this removed, thanks Andy!